instagram/src/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentSection.css';
+
+class CommentSection extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            comments: this.props.obj.comments,
+            counter: this.props.obj.likes,
+            input: ''
+        }
+    }
+
+    handleInput = (event) => {
+        const {value} = event.target;
+        this.setState({
+            input: value
+        });
+    }
+
+    addNewComment = (event) => {
+        event.preventDefault();
+        const newComment = {
+            username: "T-rex_Ninja",
+            text: this.state.input
+        }
+        this.setState({
+            comments: [...this.state.comments, newComment],
+            input: ''
+        })
+    }
+
+    addNewLike = (event) => {
+        event.preventDefault();
+        const hearts = this.state.counter;
+        this.setState({
```

Comment:
Good job implementing the like functionality! Currently it continues to increase every time it is clicked on. How would you implement it for a user, without allowing them to like it more than once? Also, here's another way to increment state without setting it to a variable 
` this.setState(prevState =>{
            return {
                likes: prevState.likes + 1, 
}`